Makefile

```diff
@@ -202,7 +202,7 @@ test-sanity: tidy build/operator-sdk lint
 
 TEST_PKGS:=$(shell go list ./... | grep -v -E 'github.com/operator-framework/operator-sdk/(hack/|test/)')
 test-unit: ## Run the unit tests
-	$(Q)go test -coverprofile=coverage.out -covermode=count -count=1 -short ${TEST_PKGS}
+	$(Q)go test -coverprofile=coverage.out -covermode=count -count=1 ${TEST_PKGS}
```

Comment:
Apparently we've never actually run any tests on the proxy because they're skipped on a short run :grimacing:

Reply:
Can you make a separate PR removing `-short` since this affects all unit tests?

Reply:
:+1: https://github.com/operator-framework/operator-sdk/pull/2718